core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -168,7 +168,7 @@
       display: none;
     }
     .oppia-exploration-footer .oppia-navbar-footer-info-icon {
-      margin-top: -11px;
+      margin-top: -1px;
     }
     .oppia-exploration-footer .oppia-navbar-footer-info-icon:hover {
       margin-top: -11px;
```

Comment:
Make this match the other value. There shouldn't be any reason for these to differ. Actually, I think you can just remove this rule entirely...

Reply:
Hi @tjiang11 , so i will make the margin top as -3px for all the conditions on the info icon (hover , or normal)   .  So will this be fine ?

Reply:
Yes, that would be fine.